modules/oe_theme_content_publication/oe_theme_content_publication.post_update.php

```diff
@@ -65,10 +66,49 @@ function oe_theme_content_publication_post_update_00004(): void {
   $display->save();
 }
 
+/**
+ * Enable Field Group and OpenEuropa Theme Content Entity Contact modules.
+ */
+function oe_theme_content_publication_post_update_00005(): void {
+  \Drupal::service('module_installer')->install(['oe_theme_content_entity_contact']);
+}
+
+/**
+ * Add Publication thumbnail image style.
+ */
+function oe_theme_content_publication_post_update_00006() {
+  // If the image style already exists, we bail out.
+  $style = \Drupal::entityTypeManager()->getStorage('image_style')->load('oe_theme_publication_thumbnail');
+  if ($style) {
+    return 'The image style was previously created.';
+  }
+
+  // Create image style.
+  $image_style = ImageStyle::create([
+    'name' => 'oe_theme_publication_thumbnail',
+    'label' => 'Publication thumbnail',
+  ]);
+
+  // Create effect.
+  $effect = [
+    'id' => 'image_scale',
+    'weight' => 1,
+    'data' => [
+      'width' => 192,
+      'height' => NULL,
```

Comment:
Keep the same settings as in the config file:
```suggestion
      'height' => 192,
```